src/agents/interactive_parameters.jl

```diff
@@ -215,10 +211,14 @@ function update_abm_data_plots!(data, axs, model, df_agent, df_model, adata, mda
     for ax in axs; autolimits!(ax); end
 end
 
-function update_abm_parameters!(model, params, slidervals)
+function update_abm_properties!(model, slidervals)
     for l in keys(slidervals)
         v = slidervals[l][]
-        model.properties[l] = v
+        if has_value(model.properties, l)
+            set_value!(model.properties, l, v)
+        else
+            throw(KeyError("$l"))
```

Comment:
Not sure if this one is needed. I thought it might be nice to point this out to the user but we could also just do that via printing instead of an error.

Reply:
This is fine.